### DISCUSSION TITLE
Copilot not working anymore after MacOS Update (12.0.1 to 12.3.1)

### DISCUSSION THREAD

**Discussion by dkbast** [community/community#16702] [Copilot Conversations] [answered]
Copilot was working just fine in nvim, but after updating my os it stopped working - it still says "Enabled and online" in the status bar but I don't get any suggestions anymore.

**Comment by davecheney** [comment, marked as answer]
Your copilot.vim plugin is out of date. Please see https://github.com/github/feedback/discussions/16423
